content/management/changelog/index.mdx

```diff
@@ -38,6 +38,13 @@ Since the migration is still ongoing, different parts will be successively moved
 
 - The [**List Group Properties**](/management/configuration-api/v3.3/#list-group-properties) method had the `group_id` parameter renamed to `id`.
 - The [**Delete Group Properties**](/management/configuration-api/v3.3/#delete-group-properties) method had the `group_id` parameter renamed to `id`.
+- Properties configuration is now owned by **Client Id**, not by **License**, which resulted in the following changes:
```

Comment:
Shouldn't we refer to **Integration** and not **Client Id**? here and in all other places

Reply:
We've been using both, I guess. Perhaps we could write Client Id (Integration).